Sources/HummingbirdHTTP2/HTTP2Channel.swift

```diff
@@ -30,26 +30,57 @@ public struct HTTP2UpgradeChannel: HTTPChannelHandler {
         public let channel: Channel
     }
 
+    /// HTTP2 Upgrade configuration
+    public struct Configuration: Sendable {
+        /// Configuration applieds to HTTP2 stream channels
+        public var streamConfiguration: HTTP1Channel.Configuration
+
+        ///  Initialize HTTP2UpgradeChannel.Configuration
+        /// - Parameters:
+        ///   - additionalChannelHandlers: Additional channel handlers to add to HTTP2 connection channel
```

Comment:
Can we document where these handlers are added?

Reply:
This is fixed in the HTTP2 PR